### PR TITLE
sec 3 `Pytorch module and Gluon blocks`

### PR DESCRIPTION
**Pull request by ZiyueHuang**
Is the initializer section suitable after block definition? @SherlockLiao

cc @szha

### CODE REVIEW

File: cheatsheets/pytorch_gluon.md [hunk 1]
```diff
@@ -50,11 +50,41 @@ MXNet uses lazy evaluation to achieve superior performance. The Python thread ju
...
+
+### usage of existing blocks look alike
+
+| Function               | PyTorch                           | MXNet Gluon                                                                |
+|------------------------|-----------------------------------|----------------------------------------------------------------------------|
+| Usage of existing blocks    |  `y=net(x)`  |  `y=net(x)`   |
+
 ### HybridBlock can be hybridized, and allows partial-shape info
+
+HybridBlock supports forwarding with both Symbol and NDArray. After hybridized, HybridBlock will create a symbolic graph representing the forward computation and cache it.
```
Comment (szha) [inline cheatsheets/pytorch_gluon.md]: It's probably worth mentioning that most of the built-in blocks (fully connected, conv, pooling, batch norm, etc.) are HybridBlocks.

Comment (ZiyueHuang) [reply cheatsheets/pytorch_gluon.md]: Added :)
